wire-schema/src/jvmMain/kotlin/com/squareup/wire/schema/CoreLoader.kt

```diff
@@ -44,7 +44,7 @@ actual object CoreLoader : Loader {
       }
     }
 
-    throw error("unexpected load: $path")
+    error("unexpected load: $path")
```

Comment:
Was this change intentional? Does this actually do anything w/out the `throw` keyword?

Reply:
It doesn't do anything haha. It creates an exception and then returns.

Reply:
<img width="368" alt="image" src="https://user-images.githubusercontent.com/7771562/217886493-ff6158c9-681a-4e80-acfd-d76d9302e419.png">

Reply:
<img width="503" alt="image" src="https://user-images.githubusercontent.com/7771562/217886570-de232c5c-ffb3-4a22-9925-ffc467a7d967.png">

Reply:
I'll remove this change